docs/support-policy.md

```diff
@@ -0,0 +1,40 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Packages support policy
+
+`tezos-packaging` provides packages for Ubuntu, Fedora and macOS for each stable and release-candidate
+version of the Octez.
+
+This document targets to explain which versions of the aforementioned OSes are targeted
+by our packages.
+
+## Ubuntu packages
+
+We provide support for all LTSes supported by Canonical:
+* 18.04 LTS (Bionic Beaver)
+* 20.04 LTS (Focal Fossa)
+
+Also, we provide support for non-LTS versions on request from the users.
+E.g. 21.04 (Hirsute Hippo) support was requested in [#212](https://github.com/serokell/tezos-packaging/issues/212).
+
+There are packages for `arm64` and `amd64` architectures.
+
+## Fedora packages
+
+<!-- TODO: mention that we support all maintained versions once TODO below is resolved -->
+We provide support for the following Fedora versions:
+* Fedora 34
+<!-- * Fedora 35 TODO: uncomment once we're done with Fedora 35 support -->
```

Comment:
Continuing the same, IMO this should mention that we try to cover all [Fedora releases](https://docs.fedoraproject.org/en-US/releases/) currently supported.

I think that it's also better to remove the TODO and instead explain that Fedora 35 is currently unsupported due to [tezos/#1449](https://gitlab.com/tezos/tezos/-/issues/1449)